web/modules/custom/error_page/markup/error_message.html

```diff
@@ -0,0 +1 @@
+<p>There was an unexpected problem serving your request. Please try again and contact us if the problem persist including <em>{{ uuid }}</em> in your message. (<a href="{{ base_path }}contact">Contact support</a>)</p>
```

Comment:
I think we will keep the Joinup styling out from this module right? In that case we shouldn't put a link to the contact form but just to the homepage. We will use a custom html template for Joinup.